addons/area_markers/XEH_preInit.sqf

```diff
@@ -23,7 +23,7 @@ if (hasInterface) then {
 
     // List of marker names that are blacklisted from being edited through Zeus
     // A marker is blacklisted if its name contains any of strings in this list
-    GVAR(blacklist) = [];
+    GVAR(blacklist) = [QEGVAR(building_markers,buildingMarker)];
```

Comment:
Do this is in the `building_markers` component - like `cover_map` component.

Reply:
https://github.com/zen-mod/ZEN/pull/704/files#diff-4334d66e7fae118d875b2d401c695aecbd7f064282b8197746d8751859ef3017R8